src/stan/services/util/experimental_message.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef STAN_SERVICES_UTIL_EXPERIMENTAL_MESSAGE_HPP
+#define STAN_SERVICES_UTIL_EXPERIMENTAL_MESSAGE_HPP
+
+#include <stan/callbacks/writer.hpp>
+
+namespace stan {
+  namespace services {
+    namespace util {
+
+      void experimental_message(stan::callbacks::writer&
```

Comment:
@syclik I'm seeing this function and thinking that it would be easier to create a namespace, say stan::message, with only static strings so that the calling function could just do writer(stan::message::experimental_procedure) rather than experimental_message(writer).  I think it would be clearer, less code, fewer calls to the writer, less formatting implied by calls to writer (such as when we call the message_writer() here to add newlines (I think that's what that's for).  It would also make it easier to check whether the string came through correctly on tests.  What do you think? I could create the strings as I write tests---there are a lot of places (e.g.in stan::services::util::initialize) where these multi-line strings are used.